.gitignore

```diff
@@ -4,7 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
-/packages/*/node_modules
+/packages/*/
```

Comment:
Why do we need this? Which files does it match?

Reply:
Sorry its a mistake from me while resolving conflicts